Felienne/hedy
# Fix incorrect CSS merge and html nested element #1242

**Description**

- Fix missing closing bracket in styles.css;
- Changed incorrect h2 element to a div  

**Fix for**

#1242

**How to test**

- The command `./build-tools/heroku/tailwind/generate-css` should run successfully without an error of a missing closing bracket
- The class-stats.html class should not contain an h2 element.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
